docker-compose.yml

```diff
@@ -1,21 +1,20 @@
-version: '3.1'
-
 # AlphaDev casinomex database
 services:
-  casinomex_db:
-    image: mysql
+  db:
+    image: mariadb
     restart: unless-stopped
     environment:
-      MYSQL_ROOT_PASSWORD: timba2024
-      MYSQL_DATABASE: timba-data-base
+      MARIADB_ROOT_PASSWORD: timba2024root
```

Comment:
**🚨 suggestion (security):** Improved environment variable setup for database credentials

The docker-compose.yml file now uses separate environment variables for the database user and password, which is a more secure and flexible approach. However, you might want to consider using external secrets management for production environments to further enhance security.